### ISSUE TITLE
disabling spelled wrong in backup-dbadatabase

### ISSUE THREAD

**Issue by potatoqualitee** [sqlcollaborative/dbatools#3890]
disabiling should be disabling

**Comment by sqllensman** [comment]
Looking at the example code for all functions there are quite a number with spelling or grammar errors.
There are also examples where the code and the description do not match. While I am happy to make the necessary minor changes should these be done via a single pull request or with separate requests per function?

**Comment by niphlod** [comment]
If they are just related to help you can issue a PR with a few functions
in. Wr wanna be agile even in layman's terms, so try to envision that for
the PR to be easily merged it should be also easy to review..
Balance the number of included files with that bit in mind (I'd say no more
than 10 on top of my head)

Il mer 5 set 2018, 09:38 Patrick Flynn <notifications@github.com> ha
scritto:
